app/models/solr_document.rb

```diff
@@ -2,27 +2,30 @@
 
 class SolrDocument
   include Blacklight::Solr::Document
-  # The following shows how to setup this blacklight document to display marc documents
-  extension_parameters[:marc_source_field] = :marc_display_ss
-  extension_parameters[:marc_format_type] = :marcxml
-  use_extension(Blacklight::Solr::Document::Marc) do |document|
-    document.key?(:marc_display_ss)
-  end
 
+  # self.unique_key = 'id'
```

Comment:
?

Reply:
seems like we do not need it, it was already there I just changed the order of lines and kept it there since I was not sure 🤔